components/input.css

```diff
@@ -0,0 +1,51 @@
+.input-container{
+  width: 100%;
+  margin: 0.5rem 1rem 0 1rem;
+  }
+  .input-label-container{
+      margin: 0.5rem 1rem 0 1rem;
```

Comment:
You can define global variables for spacing also to keep it consistent

Reply:
yes will do this when refactoring code